randovania/layout/preset_migration.py

```diff
@@ -802,12 +807,21 @@ def _migrate_v49(preset: dict) -> dict:
 
     return preset
 
+
 def _migrate_v50(preset: dict) -> dict:
     if preset["game"] == "dread":
         preset["configuration"]["raven_beak_damage_table_handling"] = "consistent_low"
     return preset
 
 
+def _migrate_v51(preset: dict) -> dict:
```

Comment:
...did we add a migration to change one-way in the preset?

Reply:
I was ignoring with the lack of use, but eh could use it